pyiron_base/jobs/job/executor.py

```diff
@@ -0,0 +1,64 @@
+from pyiron_base.jobs.job.generic import GenericJob
+
+
+class JobWithExecutor(GenericJob):
```

Comment:
Needs a docstring justifying its existence over `GenericJob`. Additionally the ability to specify the executor to be used by string needs to be documented somewhere, either here or at least on the property.

Reply:
I removed this class and implemented the functionality in the `GenericJob` class.